### PR TITLE
3. GET/api/categories - set up test suite, app, controllers and model…

### PR DESCRIPTION
**Pull request by AngelikaM-T**
…s file

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  db.end();
+});
+
+describe.only("/api/categories", () => {
+  test("GET - 200: responds with an array of category objects with slug and description as properties", () => {
+    return request(app)
+      .get("/api/categories")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.length).toBe(4);
```
Comment (JVal-NC) [inline __tests__/app.test.js:19]: Test to be revised for body length>0 rather than 4. Stops test being rigid.

File: __tests__/app.test.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        body.forEach((category) => {
+          expect(category).toEqual({
```
Comment (JVal-NC) [inline __tests__/app.test.js:21]: toEqual to be swapped for a more suited jest object matcher

File: app.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+const express = require("express");
+const app = express();
+const { getCategories } = require("./controllers/game.controller");
+
+app.use(express.json());
```
Comment (JVal-NC) [inline app.js:5]: app.use(express.json()) not needed at this stage, please remove

File: __tests__/app.test.js [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+            slug: expect.any(String),
+            description: expect.any(String),
+          });
+        });
+      });
+  });
+
+  test("GET - 404: responds with an error message, (Not found!), when passed an invalid endpoint", () => {
```
Comment (JVal-NC) [inline __tests__/app.test.js:29]: Test does not apply for /api/categories. Please can it have its own describe block

**Review by JVal-NC — COMMENTED** [review]
Hi, please can the comments be incorporated and a new pull request be made. Otherwise looks good.

**Comment by pkaodev** [comment]
Looks good, merge away!

**Pull request merged**